src/test/java/cubecart/test/BaseTest.java

```diff
@@ -1,16 +1,18 @@
 package cubecart.test;
 
-import cubecart.utils.ConfigReader;
-import cubecart.utils.Driver;
-import cubecart.utils.SessionUtil;
-import cubecart.utils.UiUtil;
+import cubecart.utils.*;
 import org.apache.log4j.Logger;
 import org.testng.annotations.*;
 
 public class BaseTest {
     SessionUtil sessionUtil = SessionUtil.getInstance();
     static Logger log = Logger.getLogger(BaseTest.class);
 
+    @BeforeClass
+    public void connectToDB() {
+        DbUtilMysql.connectToDB();
+    }
+
```

Comment:
forgot to add @AfterClass method to close db connection

Reply:
Added.